dcos_installer/upgrade.py

```diff
@@ -51,18 +51,27 @@
 SKIP_CHECKS=false
 VERBOSE=false
 
-if [[ $# -ne 0 ]]; then
-    for var in "$@"; do
-        if [[ "$var" = "--skip-checks" ]]; then
-            echo "Skipping checks"
-            SKIP_CHECKS=true
-        fi
-        if [[ "$var" = "--verbose" ]]; then
-            echo "Verbose mode on"
-            VERBOSE=true
-        fi
-    done
-fi
+while (( "$#" )); do
+  case "$1" in
+    --skip-checks)
+      shift
+      echo "Skipping checks"
+      SKIP_CHECKS=true
+      ;;
+    -v|--verbose)
+      shift
+      echo "Verbose mode on"
+      VERBOSE=true
+      ;;
+    -*|--*=) # unsupported flags
+      echo "Error: Unsupported flag $1" >&2
+      echo "$(basename "$0") [options...]
```

Comment:
Can we separate this out into a `usage()` function?

Reply:
It's only in one place so I'd like to keep it here. I started with `usage()` and `-h|--help` but then I realized we don't need it

Reply:
Alright, I just figured it's a bit easier to read